client/src/root.js

```diff
@@ -49,11 +169,16 @@ class Root extends Component {
 
         // The MainMenu component is defined here in order to avoid recreating menu structure on every change in the main menu
         // This is because Root component depends only on the language, thus it is redrawn (and the structure is recomputed) only when the language changes
+        @withComponentMixins([
+            withTranslation,
+            withForm
```

Comment:
Why does the MainMenu component has the `withForm`?

Reply:
I was testing  this because I couldn't put the NamespaceTreeSelector, but I solved and I forgot to remove this.